index.bs

```diff
@@ -729,7 +729,9 @@ An {{MLContext}} interface represents a global state of neural network execution
 
 In a situation when a GPU context executes a graph with a constant or an input in the system memory as an {{ArrayBufferView}}, the input content is automatically uploaded from the system memory to the GPU memory, and downloaded back to the system memory of an {{ArrayBufferView}} output buffer at the end of the graph execution. This data upload and download cycles will only occur whenever the execution device requires the data to be copied out of and back into the system memory, such as in the case of the GPU. It doesn't occur when the device is a CPU device. Additionally, the result of the graph execution is in a known layout format. While the execution may be optimized for a native memory access pattern in an intermediate result within the graph, the output of the last operation of the graph must convert the content back to a known layout format at the end of the graph in order to maintain the expected behavior from the caller's perspective.
 
-When an {{MLContext}} is created with {{MLContextOptions}}, the user agent selects and creates the underlying execution device by taking into account the application's {{MLPowerPreference}} and {{MLDeviceType}} options.
+When an {{MLContext}} is created with {{MLContextOptions}}, the user agent selects and creates the underlying execution device by taking into account these options, currently only the {{MLPowerPreference}} option. Depending on the underlying platform, the user agent may select different combinations of CPU, NPU and GPU devices.
```

Comment:
I think the wording
> Depending on the underlying platform, the user agent may select different combinations of CPU, NPU and GPU devices.
is superfluous in this case as it is an implementation detail for the UA

Reply:
What do you think about keeping the text, but making it a non-normative note?

Reply:
Yes, it is implementation detail, and superfluous. A note might indeed work, but if you think it's not actually needed, I'd rather remove it.

Reply:
> What do you think about keeping the text, but making it a non-normative note?

+1

Reply:
Done.